src/website/apps/event/migrations/0023_organizer_eventsideevent_eventorganizer.py

```
@@ -0,0 +1,366 @@
+    """Drop the tables (reverse of forwards)."""
+    try:
+        EventSideEvent = apps.get_model("event", "EventSideEvent")
+        EventOrganizer = apps.get_model("event", "EventOrganizer")
+        Organizer = apps.get_model("event", "Organizer")
+    except LookupError:
+        return
+    for model in (EventSideEvent, EventOrganizer, Organizer):
+        try:
+            schema_editor.delete_model(model)
+        except Exception:
+            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)